src/react/workflow/WorkflowList.tsx

```diff
@@ -1,147 +1,131 @@
-import MemoRow, { createItemData } from './WorkflowRow';
-import Table, * as T from '../ui-elements/Table';
-import { useFilters, useFlexLayout, useSortBy, useTable } from 'react-table';
-import { AutoSizer } from 'react-virtualized';
-import { FixedSizeList } from 'react-window';
-import { ListSection } from '../ui-elements/ListLayout3';
+import React, { useCallback, useEffect, useState } from 'react';
 import { TextTransformer } from '../ui-elements/Utility';
 import type { Workflows } from '../../types/workflow';
-import { useDispatch, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import { AppState } from '../../types/state';
-const columns = [
-  {
-    Header: 'Workflow Description',
-    accessor: 'name',
-    width: 60,
-  },
-  {
-    Header: 'Workflow Type',
-    accessor: 'type',
+import Table from '@scality/core-ui/dist/components/tablev2/Tablev2.component';
+import { Workflow } from '../../types/workflow';
+import { Button } from '@scality/core-ui/dist/next';
+import { TitleRow as TableHeader } from '../ui-elements/TableKeyValue';
+import { useTheme } from 'styled-components';
 
-    Cell({ value: type }: { value: string }) {
-      return <TextTransformer transform="capitalize">{type}</TextTransformer>;
-    },
+const SEARCH_QUERY_PARAM = 'search';
 
-    width: 25,
-  },
-  {
-    Header: 'State',
-    accessor: 'state',
-    Cell: ({ value }) => {
-      return value ? 'Active' : 'Inactive';
-    },
-    width: 15,
-  },
-];
 type Props = {
   workflows: Workflows;
   workflowId: string | null | undefined;
 };
 
+type RowType = {
+  id: number,
+  original: Workflow;
+  values: Workflow;
+};
+
 function WorkflowList({ workflows, workflowId }: Props) {
   const history = useHistory();
-  const dispatch = useDispatch();
-  const loading = useSelector(
-    (state: AppState) => state.networkActivity.counter > 0,
-  );
-  const {
-    getTableProps,
-    getTableBodyProps,
-    headerGroups,
-    rows,
-    setFilter,
-    prepareRow,
-  } = useTable(
+  const theme = useTheme();
+
+  function DataComponent({ row, loading, data }: {row: RowType, loading: boolean, data: string}) {
+    return loading ? (
+      <span>loading ...</span>
+    ) : (
+      <span> {`${row.values.name} ${data}`} </span>
+    );
+  }
+
+  function RowAsync({ row } : {row: RowType}) {
+    const [loading, setLoading] = useState(true);
+    const [data, setData] = useState('');
+
+    useEffect(() => {
+        setData('loading data');
```

Comment:
I think we don't need to async load the name since it already exists in the `workflows` which we pass to component WorkflowList

Reply:
Done!